src/content/guides/asset-management.md

```diff
@@ -483,6 +483,18 @@ __src/index.js__
   document.body.appendChild(component());
 ```
 
+Now once again, run your build command:
```

Comment:
We had already given example output of the build command and instruction to run the build itself, i would suggest adding this info to the next paragraph e.g.
```
Re-run the build command and open `index.html`....
```